microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -902,12 +912,54 @@ This is a programmatic way to record JavaScript execution, which can be useful i
 
 When DevTools is used to debug processes that are running in Node.js, this experiment makes whatever `console.profile` generates available to the **Performance** Tool.
 
+See also:
+* [JavaScript Profiler tool](../javascript-profiler/javascript-profiler-tool.md)
+
 Status:
-* This checkbox is present in Microsoft Edge Canary 122.
-* This checkbox is present in Microsoft Edge Stable 120.
+* This checkbox is not present in Microsoft Edge Canary 125.
+* This checkbox is present in Microsoft Edge Stable 123.
+
+
+<!-- ====================================================================== -->
+## Timeline: Enable track confirmation feature that can reorder or hide a track in the flame chart
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 125_
 
+Enables the track confirmation feature, which can reorder or hide a track in the flame chart.
+<!-- todo: need desc 
+add: in the _Foo_ tool.
+-->
+
+<!--
 See also:
-* [JavaScript Profiler tool](../javascript-profiler/javascript-profiler-tool.md)
+* []()
+-->
+
+Status:
+* This checkbox is present in Microsoft Edge Canary 125.
+* This checkbox is not present in Microsoft Edge Stable 123.
+
+
+<!-- ====================================================================== -->
+## Timeline: Enable User Timings based extensions
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 125_
+
+Enables extensions that are based on user timings.
+<!-- todo: need desc
```

Comment:
Timeline: Enable User Timings based extensions
Need real description, if possible.  What tool is this about?
@captainbrosset

Reply:
The upstream bug that introduced this experiment is also private. So I wasn't able to get much information about this.
My current understanding is this: users can install add-ons in the browser that add new tracks to the list of tracks displayed in the Performance tool. One such add-on could add a new track that displays a series of events over time, just like the Main or Interactions tracks do, but would show other types of events.

Reply:
Trying to find out more about this experiment. Will report when I know more.

Reply:
Enables extensions that customize the appearance of user timing events in the Performance tool.
from @captainbrosset offline